Silicon/CommonSocPkg/Library/BootGuardLibCBnT/BootGuardTpmEventLogLib.c

```diff
@@ -512,7 +512,14 @@ FindFitEntryData (
   UINT32                         Index;
 
   FitTableOffset = *(UINT64 *)(UINTN)(BASE_4GB - 0x40);
+  // Fit table is located in Top Swap region. Top swap region is always less than 4MB for slimboot.
```

Comment:
If we need to valid FIT, should we do it in a more common place ?  
Also , the 4MB might change later on.   If we want to limit,  I would recommend to introduce PCD to allow config.

Reply:
The limit should be in "Top Swap sizes". I will introduce a PCD to get top swap size configured from Boardconfig.
For the common FIT  functionality, I think we could plan add a separate lib as per the FIT specification later.

Reply:
Or we can consider check it in SBL build process when we generate FIT.

Reply:
Yes, We can check in SBL build process. Incase FIT pointer location is incorrect system doesn't boot.
These additional checks are added for Klocwork cases.